docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -37,6 +37,10 @@ To automatically identify a Redis Pod and monitor it with the Redis integration,
 The condition `${kubernetes.labels.app} == 'redis'` will make the {agent} look for a Pod with the  label `app:redis` within the scope defined in its manifest.
 
 For a list of provider fields that you can use in conditions, refer to <<kubernetes-provider>>.
+Some examples of conditions usage are:
+1. For a pod with label `app.kubernetes.io/name=ingress-nginx` the condition should be `condition: ${kubernetes.labels.app.kubernetes.io/name} == "ingress-nginx"`
```

Comment:
Also the numeric list here seems broken in the view https://github.com/elastic/observability-docs/blob/e7771ca322ef8a0a7b1c8cf1cc2493aa62c951d4/docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

Can you have a look to update it

Reply:
Fixed